Package.swift

```diff
@@ -115,6 +115,8 @@ func addIntegrationTestTarget(_ name: String) {
             "README.md",
             "Resources/ECSIntegTestApp/"
         ]
+    case "AWSS3":
+        additionalDependencies = ["AWSSSOAdmin"]
```

Comment:
glad we found more use cases for this!

Reply:
BTW this is not my change.  It appeared after running the package manifest generator